dask_expr/tests/test_shuffle.py

```diff
@@ -447,23 +448,36 @@ def test_sort_tail_nsmallest(df, pdf):
 @pytest.mark.parametrize("npartitions", [1, 3])
 def test_sort_values_conflicting_ascending_head_tail(pdf, npartitions):
     df = from_pandas(pdf, npartitions=npartitions)
+
+    a = df.sort_values(by=["x", "y"], ascending=[True, False]).head(10, compute=False)
```

Comment:
one more thing here: Could you assert that we didn't do a division calculation after optimise?

```
    divisions_lru.data = OrderedDict()
```

at the beginning of your test

and then 

```
assert len(divisions_lru) == 0
```

after optimise

Reply:
Done!